cve_scripts/cve_scraper.py

```diff
@@ -0,0 +1,47 @@
+import sys, getopt, os
+import wget
+import json, requests
+from selenium import webdriver
```

Comment:
Just want to confirm selenium should work headless right? and on linux?

Reply:
Yes selenium works headless. Yes it runs on linux, the user running the script needs to add the path to the chrome binary file on their laptop where I commented. I will put that in the readme that Tristan suggested making.

Reply:
@justyoyo123 did you end up adding this to config?